README.md

```diff
@@ -0,0 +1,221 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Documentation -->
+## 📽️ Project Doocumentation <a name="pro-doc"></a>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [My-Exp](https://budget-app-f5j4.onrender.com) 
+
+- [video] (https://www.loom.com/share/d12a806d77a647d7a6252a596f3a1251)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+
+In order to run this project you need:
+`Git` and `Ruby`
+```
+ gem install rails
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+```sh
+  npm install
+```
+
+Create Database (Mandatory)
+
+```
+rails db:create
+```
+
+If you have made a migration then run this command
+
+```
+rails db:migrate
+```
+```
+rails db:seed
+```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server or rails s  # after build
+``` 
+
+Run tests
+```
+rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mulugeta M.**
+
+- GitHub: [@belmeetmule](https://github.com/belmeetmule)
+- Twitter: [@belmeetmule](https://twitter.com/belmeetmule)
+- LinkedIn: [belmeetmule](https://linkedin.com/in/belmeetmule)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Use importmaps or esbuild for loading JavaScript]**
+- [ ] **[Notifications that pop up and close]**
+- [ ] **[Implement side menu and User settings]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank:
+- [Microverse](https://microverse.org)
+- Code reviewers
+- Inspiration
+   - [Vitor Guedes Madeira](https://github.com/VitorGuedesMadeira) 
+
+Under the the [Creative Commons liscense](https://creativecommons.org/licenses/by-nc/4.0/)
+
+<img width="300" alt="image" src="https://user-images.githubusercontent.com/84629565/181086933-d5bcdb09-da51-40f6-b0f8-a1f191614257.png">
+
+Original design from [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
```

Comment:
- [ ] [OPTIONAL]
Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), or optionally remove this section.